tests/integration/misc-tosca-types/runme.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+if [ $# -ne 1 ]; then
+    echo "No arguments were supplied!"
+    exit 1
+fi
```

Comment:
This is not needed when we use `set -u` at the beggining of the script.

Reply:
Great! What was I thinking not to include this :) I will immediately remove the old part of the script and rely on just `set -u`.